Tests/MapboxCoreNavigationTests/MapboxCoreNavigationTests.swift

```diff
@@ -30,8 +30,9 @@ class MapboxCoreNavigationTests: XCTestCase {
     
     override func tearDown() {
         super.tearDown()
+        navigation = nil
         UserDefaults.resetStandardUserDefaults()
-        Navigator.shared.navigator.resetRideSession()
+        Navigator._recreateNavigator()
```

Comment:
Is it safe to use this underscore method? Is it intended to be used by the SDK or is it an implementation detail subject to change without notice?

Reply:
This method was introduced only temporarily, due to the bug in the resetRideSession method.
This bug will be fixed here: https://github.com/mapbox/mapbox-navigation-native/pull/3110